JBEAP-21007 - update SSO imagestream name

luck3y opened the pull request:
https://issues.redhat.com/browse/JBEAP-21007
Signed-off-by: Ken Wills <kwills@redhat.com>

luck3y committed:
JBEAP-21007 - update SSO imagestream name

mchoma commented on templates/eap74-beta-sso-s2i.json:
```diff
@@ -46,7 +46,7 @@
             "description": "Name of the SSO image to use, examples: redhat-sso74-openshift:latest",
             "name": "SSO_IMAGE_NAME",
-            "value": "redhat-sso74-openshift:latest",
+            "value": "sso74-openshift-rhel8:7.4.5.GA",
```
Do we want to fix version to specific version?
What am I missing? I see just `latest` [1] and `7.4` [2] tags there.

[1] https://github.com/jboss-container-images/redhat-sso-7-openshift-image/blob/sso74-dev/templates/sso74-image-stream.json#L30
[2] https://github.com/jboss-container-images/redhat-sso-7-openshift-image/blob/sso74-dev/templates/sso74-image-stream.json#L37

luck3y replied:
@mchoma, thanks for spotting that -- I'd looked at their imagestreams right before doing this and I didn't see the latest tag :S
In any case, we should just use that, and assume the user has imported the latest imagestreams from whatever sso tag they want to use (we shouldn't rely on that sso -dev branch being available in the registry yet, but the tags should be.)

jmesnil committed:
[CLOUD-3839] Prepare imagestreams and templates for EAP 7.4 GA

* replace eap74-beta with eap74
* replace 7.4.0.Beta with 7.4.0
* replace jboss-eap-7-tech-preview with jboss-eap-7
* rename all files from eap74-beta-xxx to eap74-xxx
* rename the imagestreams list names to
   jboss-eap74-xxx-openshift-image-streams

JIRA: https://issues.redhat.com/browse/CLOUD-3939

Signed-off-by: Jeff Mesnil <jmesnil@redhat.com>

jmesnil committed:
[CLOUD-3943] Add metering labels for EAP 7.4

* Identify EAP 7.4 workload by adding metering lables
  to the deploymentConfig's template metadata

JIRA: https://issues.redhat.com/browse/CLOUD-3943

Signed-off-by: Jeff Mesnil <jmesnil@redhat.com>

luck3y committed:
Merge pull request #24 from jmesnil/CLOUD-3939_eap_7.4.0.GA

[CLOUD-3839] Prepare imagestreams and templates for EAP 7.4 GA

luck3y committed:
Merge pull request #26 from jmesnil/CLOUD-3943_metering_labels

[CLOUD-3943] Add metering labels for EAP 7.4
